### PR TITLE
Improve breadcrumbs across site

### PR DESCRIPTION
**Pull request by scottbommarito**
Too many to show--here's a subset:

image

image

image

image

image

### CODE REVIEW

**Comment by scottbommarito** [comment]
feedback from @anangaur 
1 - align profile icon better
2 - make breadcrumb text smaller

**Comment by skofman1** [comment]
Looks great!

File: src/NuGetGallery/Areas/Admin/Views/DeleteAccount/DeleteUserAccount.cshtml [hunk 1]
```diff
@@ -9,10 +9,7 @@
 <section role="main" class="container main-container page-delete-account">
     <div class="form-group">
         <div class="form-group">
-            <h1 class="text-center">Delete Account</h1>
-            <div class="text-center ms-font-xxl">
-                <a href="@Url.User(@Model.User)">@Model.AccountName</a>
-            </div>
+            @ViewHelpers.Breadcrumb(true, @<text><a href="@Url.User(@Model.User)">@Model.AccountName</a></text>, @<text>Delete account</text>)
```
Comment (chenriksson) [inline src/NuGetGallery/Areas/Admin/Views/DeleteAccount/DeleteUserAccount.cshtml]: Why add breadcrumb for single admin view?

If we're adding to admin views, shouldn't the breadcrumbs link back to the root admin page based on navigation?

File: src/NuGetGallery/App_Code/ViewHelpers.cshtml [hunk 2]
```diff
@@ -558,4 +558,33 @@ var hlp = new AccordionHelper(name, formModelStatePrefix, expanded, page);
             </div>
         </div>
     }
+}
+
+@helper BreadcrumbWithProfile(UrlHelper url, User currentUser, bool showDivider, params Func<MvcHtmlString, HelperResult>[] segments)
```
Comment (chenriksson) [inline src/NuGetGallery/App_Code/ViewHelpers.cshtml]: Consider compiling breadcrumbs in central location, and possibly in separate file... similar to how we do w/ UrlExtensions.

i.e., BreadCrumbs.cshtml:
```
@helper DeleteAccountBreadcrumb(Url, User) {}
@helper ManageAccountBreadcrumb(Url, User) {}
@helper ManageApiKeysBreadcrumb(Url, User) {}
@helper ManageOrganizationsBreadcrumb(Url, User) {}
@helper ManagePackagesBreadcrumb(Url, User) {}
@helper Breadcrumb(...)
```

Comment (scottbommarito) [reply src/NuGetGallery/App_Code/ViewHelpers.cshtml]: I don't see value in having a new helper for every single breadcrumb but I can see some value in having methods for similar nodes e.g. `@helper PackagesBreadcrumb` for `profile > packages > ...` and `@helper OrganizationsBreadcrumb` for `profile > organizations > ...`

File: src/NuGetGallery/App_Code/ViewHelpers.cshtml [hunk 3]
```diff
@@ -558,4 +558,33 @@ var hlp = new AccordionHelper(name, formModelStatePrefix, expanded, page);
             </div>
         </div>
     }
+}
+
+@helper BreadcrumbWithProfile(UrlHelper url, User currentUser, bool showDivider, params Func<MvcHtmlString, HelperResult>[] segments)
+{
+    var newSegments = new Func<MvcHtmlString, HelperResult>[] 
+    { 
+        @<text><a href="@url.User(currentUser.Username)" aria-label="View my profile"><i class="ms-Icon ms-Icon--Contact ms-font-l" aria-hidden="true"></i></a></text>
```
[the remark this answers is not in the document]
Comment (scottbommarito) [reply src/NuGetGallery/App_Code/ViewHelpers.cshtml]: wrapping long lines in html can sometimes lead to unwanted whitespace

File: src/NuGetGallery/ViewModels/PackageHeadingModel.cs [hunk 4]
```diff
@@ -1,11 +1,63 @@
-﻿namespace NuGetGallery
```
[the remark this answers is not in the document]
Comment (scottbommarito) [reply src/NuGetGallery/ViewModels/PackageHeadingModel.cs]: added

File: src/NuGetGallery/ViewModels/PackageHeadingModel.cs [hunk 5]
```diff
@@ -1,11 +1,63 @@
...
+            {
+                if (string.IsNullOrEmpty(Version))
+                {
+                    return Id.Abbreviate(35);
+                } else
+                {
+                    return $"{Id.Abbreviate(25)} {Version.Abbreviate(10)}";
+                }
+            }
+        }
```
[the remark this answers is not in the document]
Comment (scottbommarito) [reply src/NuGetGallery/ViewModels/PackageHeadingModel.cs]: Given this property should never be called more than once in the code, this should be fine.

**Pull request merged**